node/clients/catalog.ts

```diff
@@ -62,9 +62,11 @@ export class Catalog extends AppClient {
     )
   }
 
-  public salesChannelAvailable = (email: string) =>
+  public salesChannelAvailable = (email?: string) =>
     this.get<SalesChannelAvailable[]>(
-      `/pub/saleschannel/available?email=${encodeURIComponent(email)}`,
+      `/pub/saleschannel/available${
+        email ? `?email=${encodeURIComponent(email)}` : ''
```

Comment:
I had to read the API code.

If we don't pass the email param the API will check for all open trade policies:
https://github.com/vtex/vcs.commerce/blob/blaster/src/Catalog%20System/Vtex.Commerce.CatalogSystem.WebApi/SalesChannelWebApiPubController.cs#L84-L101

Reply:
maybe leave a comment on the code about this as well